### ISSUE TITLE
Please add USBTX & USBRX to the list of drop-down values in SerialRedirectTo block

### ISSUE THREAD

**Issue by ens-lg4** [Microsoft/pxt#1936]
Dear Developers,

I'm using your programming environment to program micro:bit in Blockly mode.

From the page https://lancaster-university.github.io/microbit-docs/ubit/serial/
it seems that the underlying Serial library supports setting the serial transmission speed on the built-in USB port in the same way it is done for a custom Serial built around non-USB pins.  If it's true, could you please also add the "USBTX" to the "TX" drop-down and "USBRX" to "RX" drop-down list?

I believe it is a similar issue to this one:  https://github.com/Microsoft/pxt/issues/1902

Many thanks in advance!

**Comment by pelikhan** [comment]
You can bundled a C++ file in your project that calls straight into mbed until we get this one in.

**Comment by ens-lg4** [comment]
@pelikhan  I don't know how to mix languages in micro:bit yet.  Can you refer me to any documentation?

( I was thinking of trying my MIDI-over-USB experiment completely in mbed/C++ environment, just to make sure that the hardware side of things works. )

How often do you usually have releases?

**Comment by pelikhan** [comment]
See https://pxt.microbit.org/packages

We try to stay on a 2 week schedule but it might take a bit more this time as some Microbit FOundation members are OOF.

**Comment by jaustin** [comment]
@pelikhan we've just had another request for this on support - would be good to to work through any issues around bringing this stuff to light.

**Comment by pelikhan** [comment]
In ``^v.0.14.1``

**Comment by jaustin** [comment]
@pelikhan the thing that's shipped looks like it's only in /beta right now?

I've used the 'redirect to USB' block, but that doesn't allow fun like setting 'rx' on the pins and 'tx' onto USB, so for example forwarding the UART from a device like a GPS sensor to the computer over USB. It does of course require both things at the same baud rate, but it's still nice to have.

I recently made this which I tested using an FTDI usb to serial adaptor: 
![image001](https://user-images.githubusercontent.com/147113/34731703-093a9128-f55b-11e7-8850-08b94655feb5.png)
https://makecode.microbit.org/_7de6D6awr8HX
(adaptor like this is good for testing http://uk.farnell.com/ftdi/ttl-232r-3v3/cable-usb-to-ttl-level-serial/dp/1329311 )

But really, we need 'USB' in this list so we can set Rx on pins and TX on USB.
<img width="442" alt="image" src="https://user-images.githubusercontent.com/147113/34731789-45432298-f55b-11e7-831c-e96319e384b3.png">

I'm not sure whether we should have 'USBTX' and 'USBRX' as separate pins, as letting people swap tx with rx over USB is odd and won't work.
